client/src/pages/Search.js

```diff
@@ -209,18 +209,18 @@ const Organizations = ({
     error,
     pageDispatchers
   })
+  // Update the total count
+  const totalCount = done
+    ? null
+    : data && data.organizationList && data.organizationList.totalCount
+  useEffect(() => setTotalCount(totalCount), [setTotalCount, totalCount])
   if (done) {
-    // Reset the total count
-    setTotalCount(null)
     return result
   }
 
   const organizations = data ? data.organizationList.list : []
-  const totalCount =
-    data && data.organizationList && data.organizationList.totalCount
-  setTotalCount(totalCount)
   if (_get(organizations, "length", 0) === 0) {
-    return <em>No organizations found</em>
+    return <em>No organizations found </em>
```

Comment:
The empty space at the end is not needed right?

Reply:
Yes; removed it again.